src/routes/user.js

```diff
@@ -14,15 +15,28 @@ userRouter.post('/register', async (req, res, next) => {
   try {
     let userRecord = await users.create(req.body);
     const output = {
-      user: userRecord,
-      userToken: userRecord.token,
+      user: userRecord
```

Comment:
Why do we not need userToken sent?

Reply:
It's in the user object so no need for the duplication.